File: app/src/main/java/com/viet/news/ui/fragment/TwoFragment.java
package com.viet.news.ui.fragment;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import com.viet.news.R;


/**
 * @author null
 */
public class TwoFragment extends LazyFragment {
    private String tabName;
    private TextView mTabNameTv;
    private View mContainerView;

    public static TwoFragment newInstance(String tabTitle) {
        TwoFragment twoFragment = new TwoFragment();
        Bundle bundle = new Bundle();
        bundle.putString("tabName", tabTitle);
        twoFragment.setArguments(bundle);
        return twoFragment;
    }

    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        tabName = getArguments().getString("tabName");
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        mContainerView = inflater.inflate(R.layout.fragment_two, container, false);
        return mContainerView;
    }

    @Override
    public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        mTabNameTv = (TextView) mContainerView.findViewById(R.id.id_fragment_two);
        mTabNameTv.setText(String.valueOf("布局2" + tabName));
        initNet();
    }

    @Override
    public void initNet() {

    }
}

File: app/src/main/java/com/viet/news/ui/fragment/TwoFragment.kt
package com.viet.news.ui.fragment

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView

import com.viet.news.R


/**
 * @author null
 */
class TwoFragment : LazyFragment() {
    private var tabName: String? = null
    private var mTabNameTv: TextView? = null
    private var mContainerView: View? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        tabName = arguments!!.getString("tabName")
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        mContainerView = inflater.inflate(R.layout.fragment_two, container, false)
        return mContainerView
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        mTabNameTv = mContainerView!!.findViewById<View>(R.id.id_fragment_two) as TextView
        mTabNameTv!!.text = "布局2${tabName!!}"
        initNet()
    }

    override fun initNet() {

    }

    companion object {

        fun newInstance(tabTitle: String): TwoFragment {
            val twoFragment = TwoFragment()
            val bundle = Bundle()
            bundle.putString("tabName", tabTitle)
            twoFragment.arguments = bundle
            return twoFragment
        }
    }
}
